### PR TITLE
[feature] add object codes on companies

### PR DESCRIPTION
**Pull request by patochectp**
ref: ND-1080

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [package]
 authors = ["Kisio Digital <team.coretools@kisio.com>", "Guillaume Pinot <texitoi@texitoi.eu>"]
 name = "transit_model"
-version = "0.31.5"
+version = "0.31.6"
```
Comment (ArnaudOggy) [inline Cargo.toml:4]: `0.32.0` maybe ?

**Comment by woshilapin** [comment]
This PR can't be merged before the specification for NTFS is modified accordingly (see https://github.com/CanalTP/ntfs-specification/pull/22). This PR will lead to the release of NTFS 0.11.4, we should also update the supported version of NTFS in `transit_model` (see [here](https://github.com/CanalTP/transit_model/blob/master/src/lib.rs#L61) and [here](https://github.com/CanalTP/transit_model/blob/master/src/ntfs/mod.rs#L397) and probably some fixtures).

File: Cargo.toml (hunk 1 above)
Comment (woshilapin) [inline Cargo.toml:4]: Should we also upgrade all the other crates of the project?

**Comment by woshilapin** [comment]
https://github.com/CanalTP/ntfs-specification/pull/22 has been merged and version 0.11.4 of the NTFS specification has been released.

**Review by woshilapin — APPROVED** [review]

File: src/gtfs/read.rs [hunk 2]
```diff
@@ -79,6 +79,7 @@ impl From<Agency> for objects::Company {
             url: Some(agency.url),
             mail: agency.email,
             phone: agency.phone,
+            codes: BTreeSet::new(),
```
Comment (ArnaudOggy) [inline src/gtfs/read.rs:82]: `..Default::default()` maybe ?
And same thing in tests in in `/src/ntfs/mods.rs`

Comment (patochectp) [reply src/gtfs/read.rs:82]: In From trait, I prefer it to be explicit.
in the tests why not @woshilapin ?

Comment (ArnaudOggy) [reply src/gtfs/read.rs:82]: It's just a detail. PR is ready to go :wink:

Comment (woshilapin) [reply src/gtfs/read.rs:82]: For one field, I'd prefer it to be explicit.

Comment (ArnaudOggy) [reply src/gtfs/read.rs:82]: Ok.So go !

**Review by ArnaudOggy — APPROVED** [review]

**Pull request merged**